### ISSUE TITLE
Using INIT_CWD as the cwd parameter in a task

### ISSUE THREAD

**Issue by mrvollger** [prefix-dev/pixi#2546]
### Problem description

Hello,

I am trying to setup a command to work outside of the project directory e.g.
```
pixi run --manifest-path ~/repos/FIRE/pixi.toml my_task
```
But I want the command to execute in my current directory and not the PIXI_PROJECT_ROOT directory.

So I tried setting up the task like this
my_task = { cmd = "snakemake -s $PIXI_PROJECT_ROOT/workflow/Snakefile", cwd = "$INIT_CWD" }

But $INIT_CWD is not a valid argument for cwd it seems:
```
× invalid working directory '$INIT_CWD'
```

Is there any way to do this?

Thanks,
Mitchell

**Comment by synapticarbors** [comment]
I'm not super familiar with `snakemake`, but would the following work?

```
my_task = { cmd = "snakemake -s $PIXI_PROJECT_ROOT/workflow/Snakefile -d $INIT_CWD" }
```

In the `snakemake` docs it says:
```
--directory, -d
Specify working directory (relative paths in the snakefile will use this as their origin).
```

**Comment by mrvollger** [comment]
Yep that works but comes with an easy pitfall for users. Any relative paths that are passed as extra arguments to snakemake will be evaluated from the pixi directory instead of the current directory. 

But you did give me an idea. I could just prefix with: `cd $INIT_CWD && ...`

Still this seems like it would be a common case for any project that has tasks you might want to launch from other directories. So I thought there might be a cleaner solution.

**Comment by mahesh-panchal** [comment]
I'm interested in this too. I use a different workflow manager (Nextflow), but there's a specific clean-up command I want to execute in any of my sub directories. It would be nice if there was a choice that a task could either be executed relative to the project root, or relative to the INIT_CWD.

**Comment by zhuoqiang** [comment]
it would great to support `$INIT_CWD` in `cwd` config for task. so instead of 

```toml
[tasks.test]
cmd = "cd $INIT_CWD && uv run pytest"
```

we could use

```toml
[tasks.test]
cmd = "uv run pytest"
cwd = "$INIT_CMD"
```

**Comment by analog-cbarber** [comment]
Probably would be good to at least document the `cmd = "cd $INIT_CWD && ..."` trick in the task documentation, since it is not immediately obvious.
